loop/payment_template.py

```diff
@@ -199,26 +199,23 @@ def excel_processing(workbook, name_of_sheets, heading_of_sheets, heading_format
                                     cell_value_from_headers.get('formula_list'),
                                     row_format)
 
-            
     except Exception as e:
         print e
     return workbook
 
 
 def prepare_value_data_generic(data):
-    data = json.loads(data)
+    if type(data) == 'str':
```

Comment:
its a patch for the time being.
To answer the question, data parameter would either be String or json object. This If condition would be executed only if data is String else Normal flow will execute